notebooks/community/model_evaluation/automl_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,1567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    print (feat_attrs_gcs_uri)
</pre><p>Please remove the spaces after&nbsp;<code>print</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>

Reply:
<p>done</p>